src/controllers/accounts/AccountsVestingInfoController.ts

```diff
@@ -0,0 +1,67 @@
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+import { IAddressParam } from 'src/types/requests';
+
+import { validateAddress } from '../../middleware';
+import { AccountsVestingInfoService } from '../../services';
+import AbstractController from '../AbstractController';
+
+/**
+ * GET vesting information for an address.
+ *
+ * Paths:
+ * - `address`: Address to query.
+ * - (Optional) `number`: Block hash or height at which to query. If not provided, queries
+ *   finalized head.
```

Comment:
Not a path, right?

Reply:
Yeah, forgot to update that part of the docs for this and the other place this happened